src/components/input/__tests__/CdrInput.spec.js

```diff
@@ -4,496 +4,489 @@ import { h } from '@vue/runtime-core';
 import CdrInput from '../CdrInput.vue';
 import CdrButton from '../../button/CdrButton.vue';
 
+const baseComponentPattern = {
+  propsData: {
+    label: 'Label Test',
+    id: 'renders',
+  }
+}
+
 describe('CdrInput', () => {
-  it('renders correctly', () => {
-    const wrapper = mount(CdrInput, {
-      propsData: {
-        label: 'Label Test',
-        id: 'renders'
-      },
-    });
-    expect(wrapper.element).toMatchSnapshot();
-  });
+  describe('basic input component', () => {
+    let wrapper;
+    let input;
+    beforeEach(() => {
+      wrapper = mount(CdrInput, { ...baseComponentPattern });
+      input = wrapper.find('.cdr-input');
```

Comment:
I don't love finding stuff by classname as a pattern because of the potential test brittleness. What do you think about targeting the input element instead?